src/components/common/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -3,22 +3,22 @@
 
     <v-card flat>
       <!-- Header -->
-      <div class="header-container">
+      <div class="header-container section-container">
```

Comment:
`<section>` or `<article>`? Just a thought.

Reply:
Definitely can do! I'm just curious on the meaning behind it

Reply:
It's giving a different name to a block of code instead of using "div" all the time. Some (maybe all) accessibility programs look at such things to present a hierarchical description of the page.

It's like in Word: you use styles for headings and stuff, right? (And on that note, it can make it easier to style articles or sections instead of misc divs.)

Reply:
If you don't have time, defer this. It's that extra 10% to get to 110% :)

Reply:
Updated!